manifests/coreos-bootimages.json

```diff
@@ -0,0 +1,16 @@
+{
+    "apiVersion": "v1",
+    "kind": "ConfigMap",
+    "metadata": {
+        "name": "coreos-bootimages",
+        "namespace": "openshift-machine-config-operator",
+        "annotations": {
+            "include.release.openshift.io/ibm-cloud-managed": "true",
+            "include.release.openshift.io/self-managed-high-availability": "true",
+            "include.release.openshift.io/single-node-developer": "true"
+        }
+    },
+    "data": {
+        "releaseVersion": "0.0.1-snapshot"
```

Comment:
Is this something that is automatically updated by the CVO or by the release tooling?

Reply:
It's replaced by `oc adm release new` yep.